server/src/main/java/org/elasticsearch/action/support/broadcast/BroadcastOperationRequestBuilder.java

```diff
@@ -24,10 +24,14 @@
 import org.elasticsearch.action.support.IndicesOptions;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public abstract class BroadcastOperationRequestBuilder<Request extends BroadcastRequest<Request>, Response extends BroadcastResponse, RequestBuilder extends BroadcastOperationRequestBuilder<Request, Response, RequestBuilder>>
-        extends ActionRequestBuilder<Request, Response, RequestBuilder> {
+public abstract class BroadcastOperationRequestBuilder<
```

Comment:
I see where the trailing "<" comes from, but it looks weird to me. Probably not much to do about it though I guess.

Reply:
Ah, just seeing this is just a backport, better leave it like in the original PR on master then.